### DISCUSSION TITLE
Storybook integration sample

### DISCUSSION THREAD

**Discussion by gabimor** [epicweb-dev/epic-stack#7] [Ideas]
I'd love to see your approch to integrating storybook and have a story for components using remix api (useLoader etc)
For some reason this is really not very clearly covered in the docs or in the community. a clear working example in your stack would be awesome

**Comment by arvigeus** [comment]
That would be nice to have. I am open to alternatives too.

**Comment by kentcdodds** [comment]
I don't know that we'll add storybook to the repo directly, but I'd welcome an example repo if someone wants to put one together!
